packages/create-redwood-app/templates/ts/web/tsconfig.json

```diff
@@ -27,7 +27,7 @@
     },
     "typeRoots": ["../node_modules/@types", "./node_modules/@types"],
     "types": ["jest", "@testing-library/jest-dom"],
-    "jsx": "preserve",
+    "jsx": "preserve"
```

Comment:
This trailing comma actually isn't valid JSON. VS Code doesn't complain but Node.js APIs do.